### PR TITLE
Add a short FAQ page answering some of the most frequent policy question we get

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

**Comment by bobholt** [comment]
@gsnedders: I'd want to be careful about the message we send here, as some of the text (e.g. "we don't want to publish any comparison tables") seems to conflict with the work we are doing on things like [wpt.fyi](http://wpt.fyi). I just had a conversation with @plehegar this morning where we talked about a possible future where the dashboard could be a "caniuse on steroids" able to provide almost line-by-line specification conformance by browser for developers in addition to our browser implementer use case.

There may need to be a broader discussion on the goals of such a dashboard and how they align with the web-platform-tests initiative as a whole.

**Review by sideshowbarker — APPROVED** [review]

**Review by sideshowbarker — APPROVED** [review]
LGTM (though acknowledging  the need to keep in mind what @bobholt noted in his comments)

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+layout: page
+title: FAQ
+---
+
+* TOC
+{:toc}
+
+## What browser does best on web-platform-tests?
+
+This is a question we've strongly avoided answering: we don't want to
```
Comment (jgraham) [inline docs/faq.md]: I think this is written in a way that draws undue attention to our cynicism :)

I think it's possible to answer the same question in a more positive way e.g.

> web-platform tests is intended to help browser vendors make more interoperable implementations. It hasn't been designed to produce metrics that are useful for simple comparisons between browsers, and as such any such comparison based on web-platform-tests results is likely to be misleading. For example, a feature might have an incomplete testsuite, leading to an inflated score, or a testsuite with a large number of tests for edge cases that are believed to have little real-world compatibility impact. Therefore making a valid interpretation of the results requires detailed knowledge of the tests and the feature under test.
>
> By avoiding scoring browsers, we hope to create an environment in which vendors feel able to release their tests that expose bugs and behavioural differences, so making it easier to find and fix such issues before they become site compatibility problems.

File: docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   pass percentage and making it closer to other vendors who passed
+   the existing tests), and
+ * Vendors delay releasing tests until they have shipped a product
+   that passes them (leading to duplication of effort and other
+   vendors fail to correctly handle some edge-cases that the delayed
+   testsuite covered).
+
+## Why can't web-platform-tests be used for [caniuse.com][]?
+
+web-platform-tests provides a large number of tests for each standard,
```
Comment (jgraham) [inline docs/faq.md]: I think this is mostly OK, but we could mention that it's essentailly a special case of the issue above.

**Review by jgraham — CHANGES_REQUESTED** [review]

**Comment by foolip** [comment]
@gsnedders, time to revive this? It's similar material to https://github.com/web-platform-tests/wpt/pull/12493, but I just put that in the appendix.

**Comment by sideshowbarker** [comment]
Can we merge this? (After rebasing and fixing the conflicts.)

**Review by stephenmcgruer — CHANGES_REQUESTED** [review]
2017 was a long time ago, and I'm not sure I agree with the answers proposed in this PR.

This could be brought as an RFC, if someone felt strongly about it. However In my view we've managed 3 years without having an FAQ to point to without significant pain (otherwise we would have landed this), so this should just be closed instead.

**Comment by sideshowbarker** [comment]
> we've managed 3 years without having an FAQ to point to without significant pain (otherwise we would have landed this), so this should just be closed instead.

Agreed — fine with this being closed (but will wait for others to weigh in, or otherwise let somebody else push the button)

File: docs/faq.md (hunk 1 above)
Comment (cvrebert) [reply docs/faq.md:11]: While the caveats remain valid, nowadays http://wpt.fyi exists and is official.

**Pull request closed without merging**